_pages/faq.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Frequently Asked Questions
+navtitle: FAQs
+styles:
+  - assets/css/faq.css
+---
+
+<details markdown="1">
+  <summary>Why is it called "Project Boise"?</summary>
+  Totally arbitrary. See the [Background](../background/) page.
+</details>
+
+<details markdown="1">
+  <summary>Are you going to "fix" ATOs?</summary>
+  No. See the [Knowns](../knowns/) page.
+</details>
+
+<details markdown="1">
+  <summary>What makes the team qualified to work on ATOs?</summary>
+  We aren't, really. At least not on our own. See the [Overview](../).
+</details>
+
+## How does Project Boise relate to...
+
+<details markdown="1">
+  <summary>FedRAMP?</summary>
+  **Both**
+
+  * Deal with software risk management
+  * Are housed within the Technology Transformation Service (TTS)
+  * Are frequently communicating with one another
+
+  **FedRAMP**
+
+  * Has been around much longer
+  * Is focused on shared cloud services
+      * Infrastructure as a Service (IaaS)
+      * Software as a Service (SaaS)
+
+  **Project Boise**
+
+  * Is doing Discovery, meaning we are in a much more exploratory phase
+  * Has a broader focus, in that we are interested in ATOs for all software, rather than just shared services
+  * Ideally will produce something useful for FedRAMP and beyond
+</details>
+
+<details markdown="1">
+  <summary>The Office of American Innovation (OAI) ATO work stream?</summary>
+  **Both**
+
+  * Started around the same time
+  * Are/were convening groups across government to talk about how to improve ATOs
+  * Are/were doing "user"/stakeholder research with agencies
+  * Are frequently communicating with one another
+  * Have an unclear future
+
+  **OAI**
+
+  * Has the convening power of the White House
+  * Isn't looking to "build"/"own" anything
+
+  **Project Boise**
+
+  * Is looking at ATOs from the perspective of "what can TTS offer?"
+  * Is talking to a broader range of stakeholders, including vendors
+</details>
```

Comment:
I agree with the idea, but I imagine both of these pages will change once we approach synthesizing, summarizing, and sharing out our discovery findings.